spec/furniture/marketplace/collecting_payments_system_spec.rb

```diff
@@ -0,0 +1,27 @@
+require "rails_helper"
+
+# @see https://github.com/zinc-collective/convene/issues/1576
+describe "Marketplace: Collecting Payments", type: :system do
+  # @see https://github.com/zinc-collective/convene/issues/1622
+  describe "via Stripe" do
+    it "Connecting a Vendor Stripe Account" do
```

Comment:
Nit: Slightly better naming here could be something like:
```suggestion
  describe "when using Stripe" do
    it "connects a Vendor Stripe Account" do
```